cumulusci/cli/logger.py

```diff
@@ -37,24 +38,17 @@ def init_logger(log_requests=False):
         requests.packages.urllib3.add_stderr_logger()
 
 
-def get_gist_logger():
-    """Determines the appropriate filepath for logfile
-    and name for the logger. Returns a logger with
-    RotatingFileHandler attached."""
-    logfile_dir = Path.home() / ".cumulusci" / "logs"
-    logfile_dir.mkdir(parents=True, exist_ok=True)
-    logfile_path = logfile_dir / "cci.log"
-
-    return get_rot_file_logger("stdout/stderr", logfile_path)
-
+def get_tempfile_logger():
+    """Creates a logger that writes to a temporary
+    logfile. Returns the logger and path to tempfile"""
+    logger = logging.getLogger("tempfile_logger")
+    temp_logfile_dir = Path.home() / ".cumulusci" / "logs"
+    temp_logfile_dir.mkdir(parents=True, exist_ok=True)
 
-def get_rot_file_logger(name, path):
-    """Returns a logger with a rotating file handler"""
-    logger = logging.getLogger(name)
+    filepath = temp_logfile_dir / f"{get_random_string(15)}.log"
```

Comment:
The temp file should probably go in the OS's preferred location for temporary files; it doesn't need to be in our final destination. I would just use `tempfile.mkstemp` to create the file and not worry about where it is or what it's named.